awx/ui/src/api/models/Root.test.js

```diff
@@ -35,10 +35,10 @@ describe('RootAPI', () => {
 
     expect(mockHttp.post).toHaveBeenCalledTimes(2);
     expect(mockHttp.post.mock.calls[0]).toContainEqual(
-      'username=foo&password=bar&next=api%2Fv2%2Fconfig%2F'
+      'username=foo&password=bar&next=%2Fapi%2Fv2%2Fconfig%2F'
```

Comment:
Does this look right?  I think it needs rework.

Reply:
Nope, its correct.  The string is encoded on purpose. The only thing we are changing is adding a / at the beginning to stop it from failing.